Eniko/index.js

```diff
@@ -0,0 +1,19 @@
+// Exercises to become familiar with data types.
+
+// Declare a variable and assign a string as a value. Make sure the string is at least 6 characters long. Print the variable to the console.
+let var1 = "tea kettle"
+console.log(var1);
+
+// Access the 2nd and 6th elements of the string. Print the elements to the console. Make sure you have selected the right index numbers
+console.log("Access the 2nd and 6th elements of the string: " + var1[2] + var1[6]);  
```

Comment:
The index of second element is 1 and index of 6th element is 5. 
console.log("Access the 2nd and 6th elements of the string: " + var1[1] + var1[5]);